apps/api/src/campaign/campaign.service.ts

```diff
@@ -78,9 +78,12 @@ export class CampaignService {
     return campaign
   }
 
-  async getCampaignByIdAndPersonId(campaignId: string, personId: string): Promise<Campaign | null> {
+  async getCampaignByIdAndCoordinatorId(
+    campaignId: string,
+    coordinatorId: string,
+  ): Promise<Campaign | null> {
     const campaign = await this.prisma.campaign.findFirst({
-      where: { id: campaignId, coordinator: { personId } },
+      where: { id: campaignId, coordinator: { personId: coordinatorId } },
```

Comment:
The `coordinatorId` is different from `personId` - two separate models and two separate tables.

We need to keep in mind that `Coordinatr` model has own field `personId` inside of it.